templates/ts-apollo-mongodb-datasync-backend/src/index.ts

```diff
@@ -26,7 +26,7 @@ async function start() {
 
   const db = await connectDB()
 
-  const { typeDefs, resolvers, contextCreator } = createDataSyncAPI(modelDefs, { db, dataSyncConflictMap:{ Comment: { enabled: true, deltaTTL: 604800 }}});
+  const { typeDefs, resolvers, contextCreator } = createDataSyncAPI(modelDefs, { db, conflictConfig: { configMap: { Comment: { enabled: true } } } });
```

Comment:
```suggestion
  const { typeDefs, resolvers, contextCreator } = createDataSyncAPI(modelDefs, { db, conflictConfig: { models: { Comment: { enabled: true } } } });
```
?

Reply:
Should it be `models` instead of `configMap`?

Reply:
Thanks for catching that!